command/v7/share_service_command_test.go

```diff
@@ -60,4 +68,67 @@ var _ = Describe("share-service Command", func() {
 		})
 	})
 
+	Context("user is logged in", func() {
+		var (
+			expectedServiceInstanceName = "fake-service-instance-name"
+			expectedSpaceName           = "fake-space-name"
+			expectedTargetedSpaceGuid   = "fake-space-guid"
+			expectedTargetedOrgGuid     = "fake-org-guid"
+		)
+
+		BeforeEach(func() {
+			cmd.RequiredArgs.ServiceInstance = expectedServiceInstanceName
+			cmd.RequiredArgs.SpaceName = expectedSpaceName
+
+			fakeSharedActor.CheckTargetReturns(nil)
+			fakeConfig.TargetedSpaceReturns(configv3.Space{GUID: expectedTargetedSpaceGuid})
+			fakeConfig.TargetedOrganizationReturns(configv3.Organization{GUID: expectedTargetedOrgGuid})
+		})
+
+		It("calls the actor to share in specified space and targeted org", func() {
+			Expect(fakeActor.ShareServiceInstanceToSpaceAndOrgCallCount()).To(Equal(1))
+
+			actualServiceInstance, actualTargetedSpace, actualTargetedOrg, actualSharingParams := fakeActor.ShareServiceInstanceToSpaceAndOrgArgsForCall(0)
+			Expect(actualServiceInstance).To(Equal(expectedServiceInstanceName))
+			Expect(actualTargetedSpace).To(Equal(expectedTargetedSpaceGuid))
+			Expect(actualTargetedOrg).To(Equal(expectedTargetedOrgGuid))
+			Expect(actualSharingParams).To(Equal(v7action.ServiceInstanceSharingParams{
+				SpaceName: expectedSpaceName,
+				OrgName:   types.OptionalString{},
+			}))
+		})
+
+		When("organization flag is specified", func() {
+			var expectedSpecifiedOrgName = "fake-org-name"
+
+			BeforeEach(func() {
+				setFlag(&cmd, "-o", flag.OptionalString{IsSet: true, Value: expectedSpecifiedOrgName})
+			})
+
+			It("calls the actor to share in specified space and org", func() {
+				Expect(fakeActor.ShareServiceInstanceToSpaceAndOrgCallCount()).To(Equal(1))
+
+				actualServiceInstance, actualTargetedSpace, actualTargetedOrg, actualSharingParams := fakeActor.ShareServiceInstanceToSpaceAndOrgArgsForCall(0)
+				Expect(actualServiceInstance).To(Equal(expectedServiceInstanceName))
+				Expect(actualTargetedSpace).To(Equal(expectedTargetedSpaceGuid))
+				Expect(actualTargetedOrg).To(Equal(expectedTargetedOrgGuid))
+				Expect(actualSharingParams).To(Equal(v7action.ServiceInstanceSharingParams{
+					SpaceName: expectedSpaceName,
+					OrgName:   types.NewOptionalString(expectedSpecifiedOrgName),
```

Comment:
This is the only difference between this `It` and the previous one. I'd delete all the other expectations have just that (personal preference; feel free to ignore)

Reply:
done